### DISCUSSION TITLE
Graphql query to get Authenticated User's Information

### DISCUSSION THREAD

**Discussion by siamekanto19** [keystonejs/keystone#7446] [Questions] [answered]
It would be pretty great if there was a Graphql Query to get the logged In User's Information, such as username, firstname, lastname, role etc. Just like having a `/users/me` route or `/auth/me` route that we usually keep in the REST API of our backend applications where user can make a GET Request with a Bearer Token to get his own information. It is really helpful for frontend. I'm not sure if it is already available in Keystone or not, if it is  then I wasn't able to find it in the Graphql Playground.

**Comment by borisno2** [comment, marked as answer]
Hey @siamekanto19,
Sounds like you are after the `authenticatedItem` query https://keystonejs.com/docs/apis/auth#graph-ql-api assuming you are using the `@keystone-6/auth` package.

**Reply by dcousens** [reply]
@siamekanto19 please unmark the answer if this is not what you were after.
